hello_nexus/basic/caller/app.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import uuid
+from typing import Optional
+
+from temporalio.client import Client
+from temporalio.worker import UnsandboxedWorkflowRunner, Worker
+
+from hello_nexus.basic.caller.workflows import CallerWorkflow
+from hello_nexus.basic.service import MyOutput
+
+NAMESPACE = "my-caller-namespace"
+TASK_QUEUE = "my-caller-task-queue"
+
+
+async def execute_caller_workflow(
+    client: Optional[Client] = None,
+) -> tuple[MyOutput, MyOutput]:
+    client = client or await Client.connect(
+        "localhost:7233",
+        namespace=NAMESPACE,
+    )
+
+    async with Worker(
+        client,
+        task_queue=TASK_QUEUE,
+        workflows=[CallerWorkflow],
+        # TODO(dan): isinstance(op, nexusrpc.contract.Operation) is failing under the
+        # sandbox in temporalio/worker/_interceptor.py
+        workflow_runner=UnsandboxedWorkflowRunner(),
```

Comment:
I think the Temporal SDK needs to auto-pass-through the entire `nexusrpc` module (which will include children) by default same as we do with `temporalio`